<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>8051simulator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>CodeArea</string>
      </property>
      <layout class="QGridLayout" name="codearea_Layout"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>908</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_newfile"/>
    <addaction name="action_openfile"/>
    <addaction name="action_savefile"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Setting</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuSetting"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_run"/>
   <addaction name="separator"/>
   <addaction name="action_debug"/>
   <addaction name="action_debugnext"/>
   <addaction name="separator"/>
   <addaction name="action_clearmemory"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_newfile">
   <property name="icon">
    <iconset>
     <normaloff>icon/new.png</normaloff>icon/new.png</iconset>
   </property>
   <property name="text">
    <string>新建文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_openfile">
   <property name="icon">
    <iconset>
     <normaloff>icon/open.png</normaloff>icon/open.png</iconset>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_savefile">
   <property name="icon">
    <iconset>
     <normaloff>icon/save.png</normaloff>icon/save.png</iconset>
   </property>
   <property name="text">
    <string>保存文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_debug">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icon/debug_off.png</normaloff>
     <normalon>icon/debug_on.png</normalon>icon/debug_off.png</iconset>
   </property>
   <property name="text">
    <string>Debug</string>
   </property>
  </action>
  <action name="action_debugnext">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icon/right_arrow.png</normaloff>icon/right_arrow.png</iconset>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </action>
  <action name="action_clearmemory">
   <property name="icon">
    <iconset>
     <normaloff>icon/clear.png</normaloff>icon/clear.png</iconset>
   </property>
   <property name="text">
    <string>ClearMemory</string>
   </property>
  </action>
  <action name="action_run">
   <property name="icon">
    <iconset>
     <normaloff>icon/start.png</normaloff>icon/start.png</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
